sdk/core/azure-core/test/ut/sha_test.cpp

```diff
@@ -7,38 +7,114 @@
 
 using namespace Azure::Core::Cryptography::_internal;
 
+// cspell: words ABCDE FGHIJ
+TEST(SHA, SHA1Test)
```

Comment:
These are some good tests.

We are missing one category of test, which is a SHA-* hash of an input, compared to a known-good expected result.

We compare 2 different hash calls return the same value, but not that it is "correct".

Something like:
We `EXPECT_EQ`
Sha1("B") = "ae4f281df5a5d0ff3cad6371f76d5c29b6d953ec"

Reply:
I think that's an excellent suggestion. However to be clear: The purpose for these newly added tests is to improve our code coverage numbers, not to verify the functionality of the algorithm.